code/modules/reagents/chemistry/holder.dm

```diff
@@ -185,6 +186,7 @@
 
 	if(transfered_by && target_atom)
 		target_atom.add_hiddenprint(transfered_by) //log prints so admins can figure out who touched it last.
+		target_atom.reagents.fingerprint_transfer = transfered_by.ckey
```

Comment:
I feel like a griefer could abuse this by having someone else add the sus regeant or add something later. It should only set this if they added the plasma

Reply:
actually it was the point but you got it lol
I will add another fingerprint check.

Reply:
I've thought about this, it is the best right now when I considered some simulations. I will tell you in discord.